docs/config.ini

```diff
@@ -1,13 +1,13 @@
 # below are the server-wide settings for documentation
 [help_settings]
-url_base = http://edx.readthedocs.org/projects/edx-partner-course-staff
-version = latest
+url_base = http://edx.readthedocs.org/projects/building-and-running-an-open-edx-course
+version = birch-release
```

Comment:
Can we call it `named-release/birch` instead? Is it possible to have a `/` in this variable?